src/ui/components/DevTools.tsx

```diff
@@ -112,9 +110,9 @@ type DevToolsProps = ConnectedProps<typeof connector>;
 const ConnectedDevTools = connector(_DevTools);
 
 const DevTools = () => (
-  <WaitForReduxSlice slice="messages">
-    <ConnectedDevTools />
-  </WaitForReduxSlice>
+  // <WaitForReduxSlice slice="messages">
```

Comment:
Without `WaitForReduxSlice`, the `DevTools` can be rendered before their setup code has finished, which may lead to a crash.

Reply:
This was a debug change. DevTools wasn't rendering so I removed this temporarily.